### ISSUE TITLE
Spilo fails to run callback_endpoint.py when KUBERNETES_USE_CONFIGMAPS is set to true

### ISSUE THREAD

**Issue by jmeth** [zalando/spilo#449]
I am running `postgres-operator:1.5.0` with the `spilo-cdp-12:1.6-p3` image for pods on an Openshift 4.4 cluster. I have `KUBERNETES_USE_CONFIGMAPS: 'true'` and `PATRONI_KUBERNETES_USE_ENDPOINTS: 'false'` set on the pods based on documentation and issues here and in the postgres-operator repo.

When the pod starts up with those configurations the `configure_spilo.py` script runs the following:
```python
    if USE_KUBERNETES:
        if placeholders.get('DCS_ENABLE_KUBERNETES_API'):
            if placeholders.get('KUBERNETES_USE_CONFIGMAPS'):
                placeholders['CALLBACK_SCRIPT'] = 'python3 /scripts/callback_endpoint.py'
```

When the pod identifies itself as a master it attempts to run the `callback_endpoint.py` script to update the master service/endpoint with the POD_IP. When doing so I get the following error message:
```sh
2020-06-18 15:19:29,292 INFO: initialized a new cluster
Traceback (most recent call last):
  File "/scripts/callback_endpoint.py", line 9, in <module>
    from kubernetes import client as k8s_client, config as k8s_config
ModuleNotFoundError: No module named 'kubernetes'
```

It appears that the spilo image is missing some sort of python3 dependency for kubernetes api and cannot update the service endpoint. Without this update the postgres service endpoint does not route traffic and the cluster initialization cannot complete. 

Am I on the right track here or is there some deeper issue that is causing this symptom?

**Comment by CyberDem0n** [comment]
Since the Patroni is using K8s API directly it already sets labels on pods and there is no need to have additional callback scripts, just the leader service need to have a label selector:
```yaml
application: spilo
version: <cluster-name>
role: master
```

**Comment by flickerfly** [comment]
Is there another variable that should be used to inform the master not to run those callback scripts? Maybe 'CALLBACK_SCRIPT=' to set it null or something?

**Comment by CyberDem0n** [comment]
No. This is a bug in spilo, but no critical. I'll fix it later.

**Comment by ReSearchITEng** [comment]
Hi @CyberDem0n,
in OCP we get some issues with , and we suspect this is the reason clusters go in SyncFailed.

PS: Also, it seems in ocp 4.4 more perms are required. This error comes AFTER we give more perms (not clear yes which are the new ones required, I gave in "bulk"). Before adding the perms, we get "ERROR: ObjectCache.run ApiException()" like in https://github.com/zalando/postgres-operator/issues/985

**Comment by CyberDem0n** [comment]
Fixed in #489
